examples/tcp.rs

```diff
@@ -0,0 +1,12 @@
+fn main () {
+    use std::net::{SocketAddr, IpAddr, Ipv4Addr, TcpStream};
+    use std::io::{Read, Write};
+
+    let addr = SocketAddr::new(IpAddr::V4(Ipv4Addr::new(31, 220, 0, 199)), 80);
```

Comment:
Could you add a comment with the human readable domain that resolves to this IP? (e.g. google.com)